games-action/another_world/another_world-0.0.1.recipe

```diff
@@ -0,0 +1,53 @@
+SUMMARY="Another World (NEO-RAW)"
+DESCRIPTION="RAW is a re-implementation of the engine used \
+in the game Another World. This game, released \
+under the name Out Of This World in non-European \
+countries, was written by Eric Chahi at the beginning of the '90s. \
```

Comment:
Remove "\" to create a linebreak. In fact, I'd insert a newline, too, so it stands out evenmore.
```suggestion
countries, was written by Eric Chahi at the beginning of the '90s. 


```